src/logger.ts

```diff
@@ -3,21 +3,48 @@ import pino = require('pino');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const pkg = require('../package.json');
 
-export type Logger = pino.Logger;
-export type LogLevels = pino.Level;
-
 const DEFAULT_LEVEL: LogLevels = (process.env.LOGLEVEL || 'info') as LogLevels;
 
-const createLogger = (level: LogLevels = DEFAULT_LEVEL): Logger =>
-  pino({
+export type Logger = pino.Logger;
+
+const createLogger = (level: LogLevels = DEFAULT_LEVEL): Logger => {
+  const pinoLogger = pino({
     level: level.toLowerCase(),
     prettyPrint: {
       messageFormat: `pact-core@${pkg.version}: {msg}`,
       translateTime: true,
     },
   });
+  pinoLogger.pactCrash = (message: string) => {
+    pinoLogger.error(pactCrashMessage(message));
+  };
+  return pinoLogger;
+};
+
+const pactCrashMessage = (
+  extraMessage: string
+) => `!!!!!!!!! PACT CRASHED !!!!!!!!!
```

Comment:
👨‍🍳  😗  (this is the closes to chef-kiss GH has...)

Reply:
It was a good idea that one @mefellows had :)